tool/tctl/common/user_command.go

```diff
@@ -493,7 +506,14 @@ func (u *UserCommand) Update(ctx context.Context, client *authclient.Client) err
 	}
 	fmt.Printf("User %v has been updated:\n", user.GetName())
 	for field, values := range updateMessages {
-		fmt.Printf("\tNew %v: %v\n", field, strings.Join(values, ","))
+		switch len(values) {
+		// Print "New xxx: null" if new value is an empty slice. Empty slice can be
+		// set by the user by passing an empty string like --set-mcp-tools "".
+		case 0:
+			fmt.Printf("\tNew %v: null\n", field)
```

Comment:
I think I'm just missing some context, but why wouldn't we not just ignore an empty input value? Is there any benefit to `--set-mcp-tools=""` ?
If this _is_ and important and relevant use case - would you not rather have a `--disable-mcp-tools` or so?